src/Application.cpp

```diff
@@ -137,6 +137,56 @@ int Application::run(QApplication &qtApp)
     this->twitch.server->connect();
 
     this->windows->getMainWindow().show();
+    if (!getSettings()->seenRecentMessagesDisclaimer)
+    {
+        auto reply = QMessageBox::question(
+            &this->windows->getMainWindow(),
+            "Do you want to enable Recent Messages support?",
+            "Recent messages is a service which allows you to see "
+            "messages from before starting Chatterino. If you enable support "
+            "for it Chatterino will send the name of every channel you have "
+            "open for the purposes of showing you the message history. You can "
+            "learn more <a "
+            "href=\"https://recent-messages.robotty.de/\">here</a> or "
+            "<a href=\"https://recent-messages.robotty.de/privacy\">view the "
```

Comment:
What if the `CHATTERINO2_RECENT_MESSAGES_URL` environment variable will be replaced?

Reply:
Changed in 78ab484, links are now dynamic based on the URL given from the ENV variable.